src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -35,6 +37,7 @@ internal class GrpcWorkerChannel : IRpcWorkerChannel, IDisposable
         private readonly string _runtime;
         private readonly IEnvironment _environment;
         private readonly IOptionsMonitor<ScriptApplicationHostOptions> _applicationHostOptions;
+        private readonly SharedMemoryManager _sharedMemoryManager;
```

Comment:
Create an interface - `IMemoryMappedFileManager` - Move all the logic that maintains and deletes list of memorymapped files into concrete implementation - `MemoryMappedFileManager`